content/hi/docs/concepts/architecture/cgroups.md

```diff
@@ -0,0 +1,128 @@
+
+cgroup v2 में प्रवास करने के लिए सुनिश्चित करें कि आप [आवश्यकताओं](#requirements) को पूरा करते हैं, और फिर उस कर्णल संस्करण पर अपग्रेड करें जिसमें cgroup v2 को डिफ़ॉल्ट रूप से सक्षम किया गया है।
+
+कुबलेट स्वचालित रूप से पहचानता है कि ऑपरेटिंग सिस्टम cgroup v2 पर चल रहा है और बिना किसी अतिरिक्त कॉन्फ़िगरेशन के यह उचित रूप से कार्रवाई करता है।
+
+cgroup v2 में स्विच करते समय उपयोगकर्ता अनुभव में कोई भी प्रमुख अंतर होना नहीं चाहिए, यहाँ तक कि यदि उपयोगकर्ता सीधे cgroup फ़ाइल सिस्टम तक पहुँच रहे हैं, या नोड से या कंटेनर के भीतर से।
+
+cgroup v2 cgroup v1 की तुलना में एक अलग API का उपयोग करता है, इसलिए यदि कोई एप्लिकेशन सीधे cgroup फ़ाइल सिस्टम तक सीधे पहुँचता है, तो उन्हें cgroup v2 को समर्थन करने वाले नए संस्करणों में अपडेट किया जाना चाहिए। उदाहरण के लिए:
+
+
+* कुछ थर्ड-पार्टी मॉनिटरिंग और सुरक्षा एजेंट्स cgroup फ़ाइल सिस्टम पर निर्भर कर सकते हैं।
+  इन एजेंट्स को cgroup v2 का समर्थन करने वाले संस्करणों में अपडेट करें।
+* यदि आप [cAdvisor](https://github.com/google/cadvisor) को स्वतंत्र
+  डेमनसेट के रूप में चला रहे हैं पॉड्स और कंटेनरों की मॉनिटरिंग के लिए, तो इसे v0.43.0 या उससे ऊपर करें।
+* यदि आप जावा एप्लिकेशन डिप्लॉय कर रहे हैं, तो पूर्णत: cgroup v2 का समर्थन करने वाले संस्करण का उपयोग करें:
+    * [OpenJDK / HotSpot](https://bugs.openjdk.org/browse/JDK-8230305): jdk8u372, 11.0.16, 15 और ऊपर
+    * [IBM Semeru Runtimes](https://www.ibm.com/support/pages/apar/IJ46681): 8.0.382.0, 11.0.20.0, 17.0.8.0, और ऊपर
+    * [IBM Java](https://www.ibm.com/support/pages/apar/IJ46681): 8.0.8.6 और ऊपर
+* यदि आप [uber-go/automaxprocs](https://github.com/uber-go/automaxprocs) पैकेज का उपयोग कर रहे हैं, सुनिश्चित करें
+  कि आप उसके उपयोग करने वाले संस्करण v1.5.1 या उससे ऊपर है।
+
+
+* कुछ थर्ड-पार्टी मॉनिटरिंग और सुरक्षा एजेंट्स cgroup फ़ाइल सिस्टम पर निर्भर कर सकते हैं। इन एजेंट्स को cgroup v2 का समर्थन करने वाले संस्करणों में अपडेट करें।
+* यदि आप [cAdvisor](https://github.com/google/cadvisor) को स्वतंत्र DaemonSet के रूप में
+  पॉड और कंटेनरों का मॉनिटरिंग के लिए चलाते हैं, तो इसे v0.43.0 या उससे बाद के संस्करण में अपडेट करें।
+* यदि आप जावा एप्लिकेशन डिप्लॉय करते हैं, तो cgroup v2 का पूरी तरह से समर्थन करने वाले संस्करणों का उपयोग करें:
+    * [OpenJDK / HotSpot](https://bugs.openjdk.org/browse/JDK-8230305): jdk8u372, 11.0.16, 15 और बाद के संस्करण
+    * [IBM Semeru Runtimes](https://www.ibm.com/support/pages/apar/IJ46681): 8.0.382.0, 11.0.20.0, 17.0.8.0, और बाद के संस्करण
+    * [IBM Java](https://www.ibm.com/support/pages/apar/IJ46681): 8.0.8.6 और बाद के संस्करण
+* यदि आप [uber-go/automaxprocs](https://github.com/uber-go/automaxprocs) पैकेज का उपयोग कर रहे हैं, तो सुनिश्चित करें
+  कि आपका उपयोग करते संस्करण v1.5.1 या उच्चतर है।
+
+
+## लिनक्स नोड्स पर cgroup संस्करण की पहचान {#check-cgroup-version}
+
+cgroup संस्करण उपयोग की जाने वाली लिनक्स वितरण और ऑपरेटिंग सिस्टम पर योग्यता पर निर्भर करता है। यह जांचने के लिए कि आपका वितरण कौनसा cgroup संस्करण उपयोग कर रहा है, नोड पर `stat -fc %T /sys/fs/cgroup/` कमांड चलाएं:
+
+
```

Comment:
At line 109 and 110  , 114 and 115 there is two empty  lines whereas  one empty line will look good.